vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputEventEmitter.cpp

```diff
@@ -109,4 +109,17 @@ void WindowsTextInputEventEmitter::onPressOut(GestureResponderEvent event) const
   });
 }
 
+void WindowsTextInputEventEmitter::onScroll(facebook::react::Point &offset) const {
```

Comment:
```suggestion
void WindowsTextInputEventEmitter::onScroll(facebook::react::Point offset) const {
```
pass by value not ref : Since Point is small (likely two floats), just pass it by value and copy into the lambda capture.